frontend/src/components/MarkdownView.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+import { markdownConverter } from '../utilities/markdown';
+
+import './MarkdownView.scss';
+
+type MarkdownViewProps = {
+  markdown?: string;
+  className?: string;
+};
+
+const MarkdownView: React.FC<MarkdownViewProps & React.HTMLAttributes<HTMLDivElement>> = ({
+  className = '',
+  markdown = '',
+  ...props
+}) => (
+  <div
+    className={classNames('oh-markdown-view', className)}
```

Comment:
would it also make sense to add something like
`word-break: break-word` so that the text gets wrapped? This way you shouldn't get a horizontal scrollbar in the drawer for long running text.